library/src/main/java/com/datatorrent/lib/io/fs/FSInputModule.java

```diff
@@ -276,7 +303,61 @@ public int getBlocksThreshold()
   {
     return blocksThreshold;
   }
-
+  
+  /**
+   * Gets minimum number of block readers for dynamic partitioning.
+   * @return minimum instances of block reader.
+   */
+  public int getMinReaders()
+  {
+    return minReaders;
+  }
+  
+  /**
+   * Sets minimum number of block readers for dynamic partitioning.
+   * @param minReaders minimum number of readers.
+   */
+  public void setMinReaders(int minReaders)
+  {
+    this.minReaders = minReaders;
+  }
+  
+  /**
+   * Gets maximum number of block readers for dynamic partitioning.
+   * @return maximum instances of block reader.
+   */
+  public int getMaxReaders()
+  {
+    return maxReaders;
+  }
+  
+  /**
+   * Sets maximum number of block readers for dynamic partitioning.
+   * @param maxReaders maximum number of readers.
+   */
+  public void setMaxReaders(int maxReaders)
+  {
+    this.maxReaders = maxReaders;
+  }
+  
+  /**
+   * Gets Interval for re-evaluating dynamic partitioning
+   * @return interval for re-evaluating dynamic partitioning
+   */
+  public long getRepartitionCheckInterval()
+  {
+    return repartitionCheckInterval;
+  }
+  
+  /**
+   * Sets Interval for re-evaluating dynamic partitioning
+   * @param repartitionCheckInterval interval for re-evaluating dynamic partitioning
+   */
+  public void setRepartitionCheckInterval(long repartitionCheckInterval)
```

Comment:
I thought so, but then I realized that interval is just to check if repartition is needed. Repartition may not always happen after this interval.

@sandeepdeshmukh  Could you suggest what would be better name in this case?